server/chat-plugins/blackjack.ts

```diff
@@ -38,22 +80,26 @@ class Blackjack extends Rooms.RoomGame {
 		};
 		this.deck = new BlackjackDeck().shuffle();
 
-		this.roomid = this.room.roomid;
+		this.roomID = this.room.roomid as RoomID;
```

Comment:
Is the `RoomID` type cast necessary? I'm almost positive that `Room#roomid`'s type is `RoomID`.